src/styles/normalize.scss

```diff
@@ -0,0 +1,21 @@
+.page {
+  font-family: $font;
+}
+
+.page__main {
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  padding: 50px 0 72px;
+}
+
+* {
+  box-sizing: border-box;
+  cursor: pointer;
```

Comment:
pointer for every item in page? what's the point? maybe apply it only for card?

Reply:
oh, now I understand. @TarasHoliuk meant only border-box for `*`, not `cursor: pointer;`. better move it back to `&--secondary` class 😉